cloud/aws/bin/deploy.py

```diff
@@ -0,0 +1,26 @@
+#! /usr/bin/env python3
+import subprocess
```

Comment:
Do you mind updating the PR description to indicate where the original versions of the bash script can be found (to make it easier to compare)?